src/Microsoft.ML.Dnn/ImageClassificationTransform.cs

```diff
@@ -338,7 +338,13 @@ private IDataView GetShuffledData(string path)
 
             ImageClassificationMetrics metrics = new ImageClassificationMetrics();
             metrics.Train = new TrainMetrics();
-            for (int epoch = 0; epoch < epochs; epoch += 1)
+            //Early stopping variables
+            bool earlyStop = false;
+            int wait = 0;
+            var history = new TrainMetrics();
+            history.Accuracy = 0;
+
```

Comment:
why are you using a TrainMetrics object when you only care about the history of the accuracy. you can simply use a float here instead.

Reply:
This was with a potential of possibly using other train metrics as well for the criteria for early stopping. I will refactor the code and this will change.